Plugins/Traccc/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+add_library(
+  ActsPluginTraccc SHARED
```

Comment:
In the end this PR should not contain any of these plugins. :thinking:

But if, for the moment, you want to test something meaningful, at least include some very basic Covfie / traccc calls in these dummy functions. To test that the compilation / linking works on at least the most basic level.

Reply:
That's the idea here.